s_tool/driver.py

```diff
@@ -49,6 +46,9 @@ def __enter__(self):
 
     def _load_driver(self):
         """Create Selenium webdriver object"""
+        # Attaching GH_TOKEN value to resolve github public api issue
+        system("export GH_TOKEN='asdasdahsdat'")
```

Comment:
remove this line from code.

the webdriver manager will read the GH_TOKEN from environment. we need to set that in environment variable.

Reply:
Sure @aahnik